src/import-data/ImportData.ts

```diff
@@ -221,7 +202,7 @@ export const ImportData = (props: ImportDataProps): ReactNode => {
   return h(Fragment, [
     h(ImportDataOverview, {
       header: getTitleForImportRequest(importRequest),
-      snapshots,
+      snapshots: importRequest.type === 'catalog-snapshots' ? importRequest.snapshots : [],
```

Comment:
I think the ImportDataOverview component is still a bit of a mess. Looking to clean that up at some point.